digits/templates/store.html

```diff
@@ -12,116 +12,153 @@
+ }
+ span.glyphicon:hover {
+     color: black;
+ }
+
+ div.progress-line-parent {
+     position: relative;
+ }
+ div.progress-line {
+     content: "";
+     position: absolute;
+     bottom: 0;
+     left: 0;
+     width: 0;
+     height: 2px;
+     background-color: rgb(51, 122, 183);
+ }
 </style>
 {% endblock %}
 
 {% block content %}
-<div id="modelList" ng-app="modelStore" ng-controller="ModelListController">
-    <div class="panel panel-default col-md-12">
-        <div class="input-group">
-          <span class="input-group-addon" id="basic-addon1">Filter list by:</span>
-          <input type="text" ng-model="query" class="form-control" placeholder="query" aria-describedby="basic-addon1">
-        </div>
-        <button type="button" class="btn btn-info" ng-click="get_model_list(1)">Update model list</button>
-        <div ng-show="groups_length==0"> No models in Model Store</div>
-        <div ng-hide="groups_length==0" ng-repeat="(key, value) in groups">
-            <div class="panel-heading">
-                <div ng-show="local_error != null">{[ local_error ]}</div>
-                <div> model store url: {[value.base_url]} </div>
+<div class="row" style="margin-bottom: 45px;">
+    <h1>Model Store</h1>
+</div>
+<div id="modelList" class="row" ng-app="modelStore" ng-controller="ModelListController">
+    <button type="button" class="btn btn-xs btn-primary" ng-click="get_model_list(1)">Update Model List</button>
+    <div ng-show="groups_length==0">
+        <h4>
+            No models in Model Store
+        </h4>
+    </div>
+    <div ng-hide="groups_length==0" class="well">
+        <!-- Filter -->
+        <div class="col-md-12">
+            <div class="pull-right">
+                <form>
+                    <div class="form-group">
+                        <div class="input-group">
+                            <div class="input-group-addon"><i class="glyphicon glyphicon-search"></i></div>
+                            <input type="text" class="form-control" placeholder="Filter" ng-model="query">
+                        </div>
+                    </div>
+                </form>
             </div>
-            <div class="panel-body">
-                <table class="table table-bordered table-hover" style="word-wrap: break-word">
-                    <thead>
-                    <tr>
-                        <th></th>
-                        <th>Name</th>
-                        <th>Contributor</th>
-                        <th>Affiliate</th>
-                        <th>Note</th>
-                        <th>Data sets</th>
-                        <th>License</th>
-                    </tr>
-                    </thead>
-                    <tbody>
-                    <tr ng-repeat="model in value.model_list | filter: query">
-                        <td>
-                            <a href="/store/push?id={[model.id]}"> Import </a>
-                        </td>
-                        <td>
+        </div>
```

Comment:
Yes.  It's not there any more.  It should be OK to merge.
@lukeyeager how do you think?